.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "overrides": [
```

Comment:
This sets Prettier (https://github.com/prettier/prettier-vscode) up to match the rules in .scss-lint.yml. Nice of you use VS Code like me.